pkg/render/guardian.go

```diff
@@ -431,25 +431,25 @@ func guardianAllowTigeraPolicy(cfg *GuardianConfiguration) (*v3.NetworkPolicy, e
 		{
 			Action:      v3.Allow,
 			Protocol:    &networkpolicy.TCPProtocol,
-			Source:      ComplianceBenchmarkerSourceEntityRule,
+			Source:      networkpolicy.CreateSourceEntityRule(ComplianceNamespace, ComplianceBenchmarkerName),
```

Comment:
I think for all of these single-tenant packages, you just want to use the `DefaultHelper` - https://github.com/Josh-Tigera/operator/blob/691931ed32299b33efcbd98c8c69d8b49f1ba760/pkg/render/common/networkpolicy/networkpolicy.go#L205

This way we still follow the same code path for single / multi-tenant code.